docs/src/main/paradox/index.md

```diff
@@ -1,273 +1,93 @@
-# Akka Distributed Workers
+# Akka Distributed Workers with Scala Guide
+ 
+Akka is a toolkit and runtime for building highly concurrent, distributed, and fault-tolerant event-driven applications on the JVM. Akka can be used with both Java and Scala.
+This guide introduces Akka by describing the Scala version of the Hello World example. If you prefer to use Akka with Java, switch to the [Akka Distributed Workers with Java guide](http://developer.lightbend.com/guides/akka-quickstart-java/).
```

Comment:
Instead of referencing Hello World: "This guide introduces Akka clusters by describing the Scala version of a distributed workers example."?